src/lib/bootloader/grub2.rb

```diff
@@ -188,87 +188,107 @@ def disk_order_summary
       else
         _(
-          "Do not install bootcode into MBR (<a href=\"enable_boot_mbr\">install</a>)"
+          "Do not write it into MBR of %s (<a href=\"enable_boot_mbr\">write</a>)"
         )
       end
+
+      format(res, Yast::BootStorage.boot_disks.map(&:name).join(", "))
     end
 
     def partition_line
-      if stage1.boot_partition?
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
+      # separated by comma
+      res = if stage1.boot_partition?
         _(
-          "Install boot code into a partition with /boot " \
-            "(<a href=\"disable_boot_boot\">do not install</a>)"
+          "Write it into partition with /boot - %s " \
+            "(<a href=\"disable_boot_boot\">do not write</a>)"
         )
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
+      # separated by comma
       else
         _(
-          "Do not install boot code into a partition with /boot " \
-            "(<a href=\"enable_boot_boot\">install</a>)"
+          "Do not write it into partition with /boot - %s " \
+            "(<a href=\"enable_boot_boot\">write</a>)"
         )
       end
+      format(res, Yast::BootStorage.boot_partitions.map(&:name).join(", "))
     end
 
     def logical_partition_line
-      if stage1.boot_partition?
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
+      # separated by comma
+      res = if stage1.boot_partition?
         _(
-          "Install boot code into a logical partition with /boot " \
-            "(<a href=\"disable_boot_boot\">do not install</a>)"
+          "Write it into logical partition with /boot - %s " \
+            "(<a href=\"disable_boot_boot\">do not write</a>)"
         )
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
+      # separated by comma
       else
         _(
-          "Do not install boot code into a logical partition with /boot " \
-            "(<a href=\"enable_boot_boot\">install</a>)"
+          "Do not write it into logical partition with /boot - %s " \
+            "(<a href=\"enable_boot_boot\">write</a>)"
         )
       end
+      format(res, Yast::BootStorage.boot_partitions.map(&:name).join(", "))
     end
 
     def extended_partition_line
-      if stage1.extended_boot_partition?
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
+      # separated by comma
+      res = if stage1.extended_boot_partition?
         _(
-          "Install boot code into a extended partition with /boot " \
-            "(<a href=\"disable_boot_extended\">do not install</a>)"
+          "Write it into extended partition with /boot - %s " \
+            "(<a href=\"disable_boot_extended\">do not write</a>)"
         )
+      # TRANSLATORS: summary line where %s is partition specified, can be more disks,
```

Comment:
To make sure the translators get this comment, shouldn't it be after the "else"?

Reply:
our pot generator use the closest comment before. so it works fine.